docs/index.rst

```diff
@@ -32,36 +32,35 @@ Welcome to the Astropy documentation! Astropy is a community-driven
 package intended to contain much of the core functionality and some common
 tools needed for performing astronomy and astrophysics with Python.
 
-.. _user-docs:
+.. _getting-started:
 
 ******************
-User Documentation
+Getting Started
 ******************
 
-.. only:: html
-
-    :doc:`whatsnew/1.3`
-    -------------------
-
-.. only:: latex
+.. toctree::
+   :maxdepth: 1
 
-    .. toctree::
-       :maxdepth: 1
+   About <http://www.astropy.org/about.html>
+   install
+   Example Gallery <generated/examples/index>
+   Tutorials <http://tutorials.astropy.org/>
+   Get Help <http://www.astropy.org/help.html>
+   Contribute and Report Problems <http://www.astropy.org/contribute.html>
```

Comment:
@MSeifert04 - sorry, I just mentioned the wrong `m` -- I meant this comment. I think it is a good suggestion.